database-couchbase/index.js

```diff
@@ -0,0 +1,43 @@
+const couchbase = require("couchbase");
+
+const cluster = new couchbase.Cluster("couchbase://localhost", {
+  username: "Administrator",
+  password: "password",
+});
```

Comment:
Since it's hard to determine if this database will be your database of choice, this might be okay for now? But I'd rather see the URI/Username/Password stored privately such as in an environment variable.

Reply:
Oof, I need to ignore that. nice catch!